### DISCUSSION TITLE
Version 6.10.0 mentioned in a commit but missing from releases

### DISCUSSION THREAD

**Discussion by zakerikk** [blockscout/blockscout#11498] [Q&A] [answered]
Hi,

I noticed that version 6.10.0 is mentioned in this commit:
https://github.com/blockscout/blockscout/commit/9e4fa7ef50a4a5b627a98aa9362fb2ab53419531

However, the latest version in the releases section is 6.9.2-beta:
https://github.com/blockscout/blockscout/releases

Has version 6.10.0 been released yet, or is it still in progress?

**Comment by vbaranov** [comment, marked as answer]
6.10.0 version is not released yet. It is tested internally. It should be published this year.

**Reply by zakerikk** [reply]
Thx
